### ISSUE TITLE
Logo contribution 

### ISSUE THREAD

**Issue by reallinfo** [wpdirectory/wpdir#8]
Hello! @PeterBooker 

I want to contribute to open source software by designing logos. If you like, I can design a logo for the restic. What do you think?

you can check my github profile to see my portfolio.

**Comment by PeterBooker** [comment]
I would definitely appreciate the contribution, a logo is not something I can do myself (well). I had a quick look and love the logos you have made, great work and it is a great way to contribute to open source projects! If you are happy designing a logo and it being used in WPdirectory go head.

You are welcome to put links to the files here or make a pull request (if you prefer) with the files added to `web/public/static/assets/` folder.

**Comment by reallinfo** [comment]
I desgined a logo. How is that? If you like it, i will send a pr. or If there's something you want in particular, I can design it.

![wp](https://user-images.githubusercontent.com/36298335/41812173-bdf11f40-7726-11e8-854f-2a7e2433036a.jpg)
![wp2](https://user-images.githubusercontent.com/36298335/41812198-64c51e3e-7727-11e8-8115-2dc8e5e1640e.jpg)

**Comment by PeterBooker** [comment]
I like the theme behind it, but it does not currently fit into the site. So perhaps we could meet half way?

I can make the header/menu a light background so it fits this sort of logo, would you be able to alter the colours slightly to fit in with the current style/colours? Could the 4 colours around the magnifying glass work around the red `#D24632` already used on the site? With the other colours mirroring that change, so slightly softer? Likewise, could the text and black on the magnifying glass match the font colour `#4a4a4a` on the site? Or if you feel that does not work I am open to suggestions.

Also, would it be possible for these to be svg files? I might be able to animate the 4 colours to spin around while searching if so, which might allow it to double as a loading icon too.

Finally, just to check, is the licensing (CC BY 4.0) is satisfied with a sentence or two (and link?) about your contribution on the about page?

**Comment by reallinfo** [comment]
Sure, I painted the colors you want. Also, I made the green color a slightly softer.  how about now? 

Btw, Licensed under the
https://creativecommons.org/licenses/by/4.0/

![1](https://user-images.githubusercontent.com/36298335/41812715-9797036e-7730-11e8-96cd-0fae076ab388.jpg)

**Comment by PeterBooker** [comment]
Perfect, I think I can make that work! I am not sure what format(s) you usually provide, but would it be possible to get the icon (without text) in SVG? It will allow me to put it to more use!

Also, I was planning to put the attribution on the about page, would you like to provide the text (and link?) to go there?

**Comment by reallinfo** [comment]
I sent a pr. https://github.com/wpdirectory/wpdir/pull/9

I added the all files version (svg and editable file) 

Also, i updated the readme.

Please check it. If there is any problem, I can fix it. Also, you can add the my github profile. Thanks!

**Comment by PeterBooker** [comment]
I have just updated the live site to use the new logos and added attributions to the [about page](https://wpdirectory.net/about). Let me know if there are any problems.

I also created some React components with the different SVG files embedded, which works really well- thank you! I will take a look at making the icon animated, so I can potentially use it as a loading icon and/or animate the logo when searches are active.
